core/src/com/unciv/logic/multiplayer/storage/MultiplayerServer.kt

```diff
@@ -48,12 +57,14 @@ class MultiplayerServer(
         SimpleHttp.sendGetRequest("${getServerUrl()}/isalive") { success, result, _ ->
             statusOk = success
             if (result.isNotEmpty()) {
-                featureSet = try {
-                    json().fromJson(ServerFeatureSet::class.java, result)
-                } catch (_: Exception) {
-                    // The server does not support server feature set - not an error!
-                    ServerFeatureSet()
-                }
+                setFeatureSet(
```

Comment:
Is there a reason to set the feature set if it's the default?

Reply:
<img width="596" height="211" alt="image" src="https://github.com/user-attachments/assets/236dc67f-7801-495a-a62f-e968960c8e5a" />

I am not sure why are we setting server feature set here. I just replaced `featureSet = ` with `setFeatureSet()` after making `featureSet` private.

Reply:
Ok, after reading the code again, I think I get what you wanted to know now. So, basically the only way to obtain `featureSet` data is do `GET /isalive`. The default value should be everything 0 since this is now Java works. So, we do need to call `setFeatureSet()` and update the initial value whenever we get new data.

It is important that we specifically call  `setFeatureSet()` also. Since, `chatVersion` is initially `0`, if we do not call it, `ChatButton` remains disabled and WebSocket never starts. So, this is as intended.